node/src/actors/node.rs

```diff
@@ -62,7 +62,9 @@ pub fn run(config: Config, callback: fn()) -> Result<(), failure::Error> {
     SystemRegistry::set(json_rpc_server_addr);
 
     // Run system
```

Comment:
Maybe add a short comment to the code explaining the change?

```suggestion
    // Run system inside a future
    // This is believed to help with issue #1088
```

Reply:
Applied!

Reply:
The important line is the one that references the issue, and I think it got lost?

Reply:
Don't worry, thanks for sending it once again. Updated!